libtextsecure/sendmessage.js

```diff
@@ -411,7 +411,7 @@ MessageSender.prototype = {
 
     const ourNumber = textsecure.storage.user.getNumber();
 
-    numbers.forEach(number => {
+    numbers.forEach(async number => {
```

Comment:
maybe note that the forEach is not going to wait for all these to be executable in case someone tries to add code after

Reply:
Yep i know, i'll add a comment in above. The reason i added `async` in there was because the `forEach` was not dependant on the other results

Reply:
yea, it looked fine just trying to look out for our future selves, though it is also tough to see us actually adding anything after though, the send is really the goal. So probably fine.